chessgame/src/common/pieces/King.ts

```diff
@@ -0,0 +1,26 @@
+import Piece from "./_Interface";
+import tails from "../constant/tails";
+
+class King extends Piece {
+    type: string = "King";
+
+    canMove(column: string, row: number): boolean {
+        const ccn: number = tails[this.column];     //current column number
+        const ncn: number = tails[column];          //new column number
```

Comment:
Tu bym zrobil zmienne columnDifference i rowDifference od razu, to ladniej bedzie widac o co chodzi i nie trzeba bedzie powtarzac tego Math.abs ciagle